junit-platform-commons/src/main/java/org/junit/platform/commons/util/AnnotationUtils.java

```diff
@@ -228,7 +228,7 @@ else if (candidate.annotationType().equals(containerType)) {
 					// Note: it's not a legitimate container annotation if it doesn't declare
 					// a 'value' attribute that returns an array of the contained annotation type.
 					// Thus we proceed without verifying this assumption.
-					Method method = ReflectionUtils.getMethod(containerType, "value").get();
+					Method method = ReflectionUtils.getMethod(containerType, "value").orElseThrow(AssertionError::new);
```

Comment:
Since we are throwing our own exception here instead of the default NoSuchElementException from Optional::get, wouldn't it be better if we added a message detailing the assumption that failed?

something like "container must declare a 'value' attribute"